src/views/InvolvementsAll/index.jsx

```diff
@@ -173,86 +169,104 @@ const InvolvementsAll = () => {
-      )}
-
-      {/* All Involvements (public) */}
-      <Grid item xs={12} lg={8}>
-        <Card>
-          <CardHeader
-            title={`${involvementSessionText} Involvements`}
-            style={{
-              backgroundColor: gordonColors.primary.blue,
-              color: gordonColors.neutral.grayShades[50],
-            }}
-          />
-          <CardContent>
-            {loading ? (
-              <GordonLoader />
-            ) : (
-              <InvolvementsGrid
-                involvements={involvements}
-                sessionCode={selectedSession}
-                noInvolvementsText="There aren't any involvements for the selected session and type"
-              />
-            )}
+              </TabPanel>
+              <TabPanel value="2">
+                <Grid
+                  item
+                  xs={12}
+                  lg={12}
+                  xl={12}
+                  container
+                  alignItems="center"
+                  justifyContent="center"
+                >
+                  {!isOnline ? null : loadingProfile ? (
+                    <GordonLoader />
+                  ) : (
+                    profile && <Requests profile={profile} session={selectedSession} />
+                  )}
+                </Grid>
+              </TabPanel>
+              <TabPanel value="3">
+                <Grid item xs={12} lg={12}>
+                  <Grid container spacing={2}>
+                    <Grid item xs={12} lg={6} marginBottom={4}>
+                      <TextField
+                        id="search"
+                        variant="filled"
+                        label="Search"
+                        type="search"
+                        value={search}
+                        onChange={(event) => setSearch(event.target.value)}
+                        fullWidth
+                      />
+                    </Grid>
+                    <Grid item xs={12} md={6} lg={6} marginBottom={4}>
+                      <FormControl fullWidth variant="filled">
+                        <InputLabel id="activity-type">Type</InputLabel>
+                        <Select
+                          labelId="activity-type"
+                          id="activity-type"
+                          value={type}
+                          onChange={(event) => setType(event.target.value)}
+                        >
+                          <MenuItem label="All" value="">
+                            <em>All</em>
+                          </MenuItem>
+                          {types.map((type) => (
+                            <MenuItem value={type} key={type}>
+                              {type}
+                            </MenuItem>
+                          ))}
+                        </Select>
+                      </FormControl>
+                    </Grid>
+                  </Grid>
+                  {loading ? (
+                    <GordonLoader />
+                  ) : (
```

Comment:
If you follow my instructions for combining `loading` and `loadingProfile` from above, you can remove this logic.